api/axis.py

```diff
@@ -443,13 +456,17 @@ def process_one(self, e, namespaces):
             expiration_date = expiration_date.text
             expiration_date = datetime.strptime(
                 expiration_date, self.FULL_DATE_FORMAT)
-            
+
+        # WTH??? Why is identifier None? Is this ever used?
         fulfillment = FulfillmentInfo(
+            collection=None, data_source_name=DataSource.AXIS_360,
```

Comment:
It looks like you (or I) added `collection` to this class's constructor, so now we can set collection on all the *Info objects created in this class without relying on the calling code to do it.